cmd/mesh/manifest.go

```diff
@@ -89,6 +93,27 @@ func genManifests(inFilename string, setOverlayYAML string) (name.ManifestMap, e
 	if err != nil {
 		return nil, err
 	}
+	// for installation path contains version number only
+	isp := mergedICPS.InstallPackagePath
+	if version.IsVersionString(isp) {
```

Comment:
you can assume version string is ok, since we pass isp through validation. this should just check if it set to anything.

Reply:
The goal here is not for validation. The problem is that there might be different kinds of install package path being set, e.g. local filepath, http url, version string only, here I want to construct the full url for version string only case.

Reply:
discussed offline, update this part